tests/models/test_cpu.py

```diff
@@ -98,10 +98,12 @@ def test_early_stopping_cpu_model(tmpdir):
 
     class ModelTrainVal(BoringModel):
 
-        def validation_epoch_end(self, outputs) -> None:
-            val_loss = torch.stack([x["x"] for x in outputs]).mean()
-            self.log('val_loss', val_loss)
+        def validation_step(self, *args, **kwargs):
+            output = super().validation_step(*args, **kwargs)
+            self.log('val_loss', output['x'])
```

Comment:
Do we need `on_epoch` here to make it consistent with previous behavior?

Reply:
for `validation_step`, `on_step=False` and `on_epoch=True` by default.